snoop/data/digests.py

```diff
@@ -142,15 +130,40 @@ def gather(blob, **depends_on):
             rv['location'] = exif_data.get('location')
             rv['date-created'] = exif_data.get('date-created')
 
+    digest, _ = models.Digest.objects.get_or_create(blob=blob)
+    rv = defaultdict([], rv)
```

Comment:
why the copy here? who's gonna reference the old one, anyway?

Reply:
I've casted `rv` into a `defaultdict` in order to append to the entities if entities are found in the text as well as in the OCR results. But since I've put all that into the entities file, I only need the `defaultdict` in there now.